src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.model;
+
+import java.util.Objects;
+
+public class Car {
+
+    public static final int RANDOM_THRESHOLD = 4;
+
+    private int position;
+
+    public void moveForward(int randomNumber) {
+        if (movable(randomNumber)) {
+            position++;
+        }
+    }
+
+    private static boolean movable(int randomNumber) {
```

Comment:
```suggestion
    private boolean movable(int randomNumber) {
```
static 제거되어도 될 것 같아요 :)